Makefile

```diff
@@ -95,3 +97,31 @@ check-inside-toolbox:
 
 requre-purge-files:
 	pre-commit run requre-purge --all-files --verbose --hook-stage manual
+
+# run all the pods needed by the service pod
+# use docker-compose to update and run them all
+# stop the service pod to be able to re-run it
+# later with different options
+compose-for-migrate-db:
+	$(COMPOSE) up --build --force-recreate -d service
+	$(COMPOSE) stop service
```

Comment:
The comment says: "stop the service pod to be able to re-run it later with different options", but in the `migrate-db`, we don't run the (docker-compose service) 'service', but a new podman container/pod, which I don't see how it's related to the previously up-ed & stop-ed 'service'.

If the purpose of this `compose-for-migrate-db` target is to make sure the `quay.io/packit/packit-service:dev` image is built and up-to-date, then I think `$(COMPOSE) build service` should be enough.

Reply:
> The comment says: "stop the service pod to be able to re-run it later with different options", but in the `migrate-db`, we don't run the (docker-compose service) 'service', but a new podman container/pod, which I don't see how it's related to the previously up-ed & stop-ed 'service'.

You are right, they are different pods and I need to fix the documentation, but since I am mapping the same ports (and maybe this is not necessary) for this reason I am closing the one created by docker-compose.

> If the purpose of this `compose-for-migrate-db` target is to make sure the `quay.io/packit/packit-service:dev` image is built and up-to-date, then I think `$(COMPOSE) build service` should be enough.

I need also the postgres (and probably the redis) pod; the service pod depends on them and I do not want to hard code this dependency here again. This is the reason why I used docker-compose to start them all.